modules/ROOT/pages/create-tokenization-service.adoc

```diff
@@ -0,0 +1,37 @@
+= Create a Tokenization Service
+
+The tokenization service enables you to substitute a sensitive data element with a non-sensitive equivalent.
+
+== Prerequisites
+
+To configure and use the tokenization service, you need the following:
+
+* Runtime Fabric 1.1.153 or later with inbound traffic configured. See xref:1.2@runtime-fabric::index.adoc[the Runtime Fabric documentation].
+* A secret group to store the tokenization table encryption keys. The tokenization service creates and stores its own keys. See xref:asm-secret-group-concept.adoc[Secrets Manager documentation].
+* A tokenization format, which describes the format and how data is tokenized. See xref:tokenization-formats.adoc[Tokenization Formats].
+
+== Create the Tokenization Service
+
+. Go to the *Runtime Manager­>Tokenization Service* page to create the service.
+. Click *Create Tokenization Service*. The prerequisites you completed provide the information you need to fill out the form. This example shows the form filled in with the following values: +
+  * In the *Runtime fabric* drop-down list, select the Runtime Fabric to deploy the tokenization service to.
+  * Select the tokenization format from the *Formats* drop-down list. You can assign more than one, or all, formats to one tokenization service.
+  * Select the secret group from the *Secret Group* drop-down list.
+  * Select the number of tokenization service replicas to run. The tokenization service runs on worker nodes in Runtime Fabric.
+  * Optionally, you can adjust the logging level for the tokenization service.
+. Click *Build and Deploy* to create the tokenization table. +
+A mapping table, which contains a large table of randomizations that are used at the core of tokenizing and detokenizing pre-builds. This is not a one-to-one table of mappings--it is used during internal steps to swap in and swap out randomizations in place of the actual data. The actual algorithm consists of these three steps:
+  ** First stage transformation: Perform AES­FFX[radix] specific domain encryption using first stage symmetric key and tweak. This will redistribute the information contained within the data to be tokenized across all characters of the extracted data.
+  ** Second stage transformation: Look­up and replace chunks of the clear­text data with the appropriate precomputed randomization in the mapping table.
+  ** Third stage transformation: Perform AES­FFX[radix] specific domain encryption using third stage symmetric key and tweak. This will redistribute the information mapped by the multiple table look­ups of the preceding step across all of the extracted.
+
+The tokenization mapping table build is a one-time action and can take awhile to build, depending on the size of the tokenization format. For example, an SSN-only table less than 200 MB in size can build in two minutes, but a larger format such as “lax alphanumeric” might take up to 20 minutes to build.
```

Comment:
You called it a table creation in line 22. Use build or create in both 22 and here?
I'd be tempted to delete "an can take awhile to build" (since it's colloquial), and just say "...one-time action. The time needed to build this table depends on the size of the tokenization format. For example,..."